iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/AbstractAcDcConverter.java

```diff
@@ -76,27 +76,27 @@ public Optional<Terminal> getTerminal2() {
     }
 
     @Override
-    public TwoSides getSide(Terminal terminal) {
+    public TerminalNumber getTerminalNumber(Terminal terminal) {
         Objects.requireNonNull(terminal);
         if (getTerminal1() == terminal) {
-            return TwoSides.ONE;
+            return TerminalNumber.ONE;
         } else if (getTerminal2().orElse(null) == terminal) {
-            return TwoSides.TWO;
+            return TerminalNumber.TWO;
         } else {
             throw new IllegalStateException("The terminal is not connected to this AC/DC converter");
         }
     }
 
     @Override
-    public Terminal getTerminal(TwoSides side) {
-        Objects.requireNonNull(side);
+    public Terminal getTerminal(TerminalNumber terminalNumber) {
```

Comment:
Throwing an exception if terminal 2 does not exist implies that the user will always know in advance if the object has one or two terminals. Maybe this method should return an optional, to make explicit that not always both terminals exist ?

Reply:
Changed to return an optional, let me know